Sources/StructuredQueries/Macros.swift

```diff
@@ -1,6 +1,6 @@
 import StructuredQueriesCore
 
-/// Defines and implements a conformance to the ``Table`` protocol.
+/// Defines and implements a conformance to the ``/StructuredQueriesCore/Table`` protocol.
```

Comment:
StructuredQueries can now link to symbols in StructuredQueriesCore, but you need to start with a root slash.